doc/cql3/CQL.textile

```diff
@@ -2552,8 +2560,8 @@ CQL distinguishes between _reserved_ and _non-reserved_ keywords. Reserved keywo
 | @VIEW@         | yes |
 | @WHERE@        | yes |
 | @WITH@         | yes |
-| @WRITETIME@    | no  |
-| @MAXWRITETIME@    | no  |
+| @WRITETIME@    | yes |
```

Comment:
Is this change unrelated to this PR/ticket? just for my understanding

Reply:
It's unrelated, I fixed the alignement because the `WITHOUT` keyword is next to it without the additional, optional change mentioned [here](https://issues.apache.org/jira/browse/CASSANDRA-18068?focusedCommentId=17682967&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17682967).

Reply:
But why writetime changes from being non reserved to being reserved? Sorry if I am being thick.

Reply:
Oh, right, I was thinking on `MAXWRITETIME ` instead of `WRITETIME `. Fixed.